pytest chokes on `test_appenv.test_empty_crontabs`

## Description

pytest chokes on the `test_appenv.test_empty_crontabs` cases added in #6780:

```
    ==================================== ERRORS ====================================
    ________________ ERROR collecting testinfra/app/test_appenv.py _________________
    ../testinfra/app/test_appenv.py:108: in <module>
        @pytest.mark.parameterize("user", ("root", "www-data"))
    E   Failed: Unknown 'parameterize' mark, did you mean 'parametrize'?
    ________________ ERROR collecting testinfra/app/test_appenv.py _________________
    ../testinfra/app/test_appenv.py:108: in <module>
        @pytest.mark.parameterize("user", ("root", "www-data"))
    E   Failed: Unknown 'parameterize' mark, did you mean 'parametrize'?
```

## Steps to Reproduce

https://app.circleci.com/pipelines/github/freedomofpress/securedrop/5807/workflows/7f4319aa-93b6-4ffb-a415-850f40343dae/jobs/68893

## Expected Behavior

Passes.

## Actual Behavior

Fails.

## Comments

CI *did* pass on #6780.  But because that branch didn't have the magic `stg-` prefix, `staging-test-with-rebase` didn't run, which means these tests didn't run either.  We don't have a way to enforce that.  Mea culpa.

--- a/molecule/testinfra/app/test_appenv.py
+++ b/molecule/testinfra/app/test_appenv.py
@@ -105,7 +105,7 @@ def test_ensure_logo(host):
         assert f.group == "root"
 
 
-@pytest.mark.parameterize("user", ("root", "www-data"))
+@pytest.mark.parametrize("user", ("root", "www-data"))
 def test_empty_crontabs(host, user):
     """Ensure root + www-data crontabs are empty"""
     with host.sudo():

